app/views/govuk_publishing_components/components/docs/attachment.yml

```diff
@@ -0,0 +1,53 @@
+name: Attachment (experimental)
+description: Displays a link to download an attachment and metadata about the file
+body: |
+  This component is marked as experimental as it is part of a drive to provide
+  a consistent place for attachment presentation logic (currently this logic is
+  within [Govspeak][] and [Whitehall][]). The API for this
+  component may change during this process.
+
+  It is used to show data about an attachment document to GOV.UK content.
```

Comment:
Not sure if I wrote this or not but feel like this sentence could be improved with a bit more information on what this is and it's usages.

Reply:
I've added a bit more to this documentation - let me know if that's ok or if there's anything else you think needs to be added.